src/validation/rules/KnownArgumentNamesRule.ts

```diff
@@ -26,15 +26,14 @@ export function KnownArgumentNamesRule(context: ValidationContext): ASTVisitor {
     Argument(argNode) {
       const argDef = context.getArgument();
       const fieldDef = context.getFieldDef();
-      const parentType = context.getParentType();
 
-      if (!argDef && fieldDef && parentType) {
```

Comment:
oh that is a much cleaner pattern: it's great that we can now remove the only-needed-for-the-message value.